test/edge/kernel_runtime_context.h

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+
```

Comment:
I think all files in pytorch/pytorch are covered by some license and we don't need this header

Reply:
Ah I was just copying the latest version of the internal header to here, and this came along. I can remove it.